alf/experience_replayers/replay_buffer.py

```diff
@@ -296,7 +267,7 @@ def get_batch(self, batch_size, batch_length):
 
             start_idx = idx
             info = BatchInfo(
-                env_ids=env_ids, positions=self._pad(idx, env_ids))
+                env_ids=env_ids, positions=self._pad(start_idx, env_ids))
```

Comment:
return `BatchInfo` directly from _prioritized_sample() and _uniform_sample() so that we can avoid redundant _pad() and circular()